src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Ball ball = new Ball();

```

Comment:
Consider making 'ball' a local variable inside the 'getRandomBall' method instead of a class field, as it's only used in that method and doesn't need to be shared across different method calls.